volumes/vds/py/vds_functions.py

```diff
@@ -0,0 +1,352 @@
+import logging
+
+LOGGER = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+def pull_raw_vdsdata(rds_conn, itsc_conn, start_date):
+#pulls data from ITSC table `vdsdata` and inserts into RDS table `vds.raw_vdsdata`
+
+    import pandas as pd
+    from numpy import nan
+    from psycopg2 import sql, Error
+    from psycopg2.extras import execute_values
+
+    # Pull raw data from Postgres database
+    raw_sql = sql.SQL('''SELECT 
+        d.divisionid,
+        d.vdsid,
+        d.timestamputc, --timestamp in INTEGER (UTC)
+        d.lanedata
+    FROM public.vdsdata AS d
+    WHERE
+        timestamputc >= extract(epoch from timestamp with time zone {start})
+        AND timestamputc < extract(epoch from timestamp with time zone {start} + INTERVAL '1 DAY')
+        AND d.divisionid = 2; --other is 8001 which are traffic signal detectors and are mostly empty.
+    ''').format(
+        start = sql.Literal(start_date + ' 00:00:00 EST5EDT')
+    )
+
+    try: 
+        with itsc_conn.get_conn() as con:
+            raw_data = pd.read_sql(raw_sql, con)
+            LOGGER.info('Fetching vdsdata')
```

Comment:
move before `pd.read_sql` to let people know that it's reading the data (in case that takes some time to execute)